packages/emitter-framework/src/typescript/components/interface-declaration.tsx

```diff
@@ -144,10 +131,13 @@ function membersFromType(type: Model | Interface): Children {
   }
 
   return (
-    <ay.For each={Array.from(typeMembers.entries())} line>
-      {([_, prop]) => {
-        return <InterfaceMember type={prop} />;
-      }}
-    </ay.For>
+    <>
+      <ay.For each={Array.from(typeMembers.entries())} line ender=";">
+        {([_, prop]) => {
+          return <InterfaceMember type={prop} />;
```

Comment:
TODO: pass a refkey? What would be an applicable refkey?

Reply:
I don't think we can make one intended for users, which is an interesting limitation of using this component. I think for now if you need to reference the interface members you'll have to use InterfaceMember manually.